x-pack/plugins/monitoring/server/alerts/base_alert.ts

```diff
@@ -286,7 +285,10 @@ export class BaseAlert {
       // for each node, update the alert's state with node state
       for (const node of nodes) {
         const newAlertStates: AlertNodeState[] = [];
-        const instance = services.alertInstanceFactory(node.meta.nodeId || node.meta.instanceId);
+        // quick fix for now so that non node level alerts will use the cluster id
+        const instance = services.alertInstanceFactory(
+          node.meta.nodeId || node.meta.instanceId || cluster.clusterUuid
```

Comment:
Some alerts are cluster level and do not have a nodeId or instanceId property, so use the clusterUuid.  This should be typed and improved at some point or have a different function for cluster vs node level processing.